project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
 addSbtPlugin("com.github.sbt" % "sbt-github-actions" % "0.25.0")
-addSbtPlugin("com.github.sbt" % "sbt-ci-release"     % "1.9.3")
+addSbtPlugin("com.github.sbt" % "sbt-ci-release"     % "1.11.0")
```

Comment:
I'm in the process of migrating com.github.sbt to the Central Portal. Please update to sbt-ci-release 1.11.0 only after that's done.